x-pack/plugins/profiling/public/components/stacked_bar_chart.tsx

```diff
@@ -87,14 +89,16 @@ export const StackedBarChart: React.FC<StackedBarChartProps> = ({
 
   const { chartsBaseTheme, chartsTheme } = useProfilingChartsTheme();
 
-  const customTooltip = highlightedSubchart
-    ? (props: TooltipInfo) => (
-        <SubchartTooltip
-          {...props}
-          highlightedSubchart={highlightedSubchart!}
-          highlightedSample={highlightedSample}
-          showFrames={showFrames}
-        />
+  const customTooltip: CustomTooltip = highlightedSubchart
+    ? (props) => (
+        <TooltipContainer>
+          <SubchartTooltip
```

Comment:
Are we using the tooltip container here to keep the rounder border?

Reply:
Let's see what O11y folks think about that

Reply:
Perhaps I missed it, but I didn't notice any difference with the borders. However, I did notice that the numeric value in the top left of the tooltip chart increased in size when the tooltip container was removed (see the screenshots below). My preference is to keep the tooltip container for now.

#### `@elastic/charts v52.0.0`

![image](https://user-images.githubusercontent.com/6038/219461500-7e259952-fbe4-4261-9f22-356c34472d29.png)

#### `@elastic/charts v53.1.0` with `TooltipContainer`

![image](https://user-images.githubusercontent.com/6038/219458636-d5f77ad1-6aaa-497a-80b1-af7585b50cfa.png)

#### `@elastic/charts v53.1.0` without `TooltipContainer`

![image](https://user-images.githubusercontent.com/6038/219458611-65a71208-be29-446d-9cf1-f924e17589c3.png)

Reply:
Perfect thanks